packages/cli/src/lib/commands/init/utils.ts

```diff
@@ -1,6 +1,17 @@
-import { CollectRcOptions, PersistRcOptions } from '../collect/options/types';
+import { CollectRcOptions, PersistRcOptions, ReportFormat } from '../collect/options/types';
 import { InitOptions } from './options';
 import { AssertRcOptions } from '../assert/options';
+import { REPORT_FORMAT_VALUES } from '../collect/constants';
+
+const isValidFormat = (value: any): value is ReportFormat => REPORT_FORMAT_VALUES.includes(value);
+
+function sanitizedFormats(formats: string[]): ReportFormat[] {
+  const validatedFormats: ReportFormat[] = formats.filter(isValidFormat);
```

Comment:
The type is inferred here.

```suggestion
  const validatedFormats = formats.filter(isValidFormat);
```